smallrye-reactive-messaging-provider/pom.xml

```diff
@@ -13,13 +13,18 @@
 
   <dependencies>
     <dependency>
-      <groupId>org.jboss</groupId>
-      <artifactId>jandex</artifactId>
+      <groupId>io.smallrye.reactive</groupId>
+      <artifactId>smallrye-reactive-messaging-api</artifactId>
     </dependency>
 
     <dependency>
-      <groupId>javax.annotation</groupId>
-      <artifactId>javax.annotation-api</artifactId>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
```

Comment:
Any reason for this dependency?

Reply:
`MediatorConfigurationSupport` uses `ClassUtils` from it.

Reply:
We should consider removing it at some point. Depending on the lib for a single class, does not make a lot of sense.